Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM python:alpine AS gen_schemas
+RUN pip install --no-cache-dir PyYAML
+ADD . /userdb
+WORKDIR  /userdb
+RUN /userdb/json-schemas.py
+
+FROM alpine:latest
+RUN apk --no-cache add make postgresql-client
+ADD . /userdb
+COPY --from=gen_schemas /userdb/json-schemas.sql.tmp /userdb/json-schemas.sql.tmp
+WORKDIR  /userdb
```

Comment:
When you drop into the user, you can set the workdir there. Moving this unchanging layer higher up